state/state.go

```diff
@@ -666,14 +694,16 @@ func (s *State) Messages(channelID discord.ChannelID) ([]discord.Message, error)
 
 // Presence checks the state for user presences. If no guildID is given, it
 // will look for the presence in all cached guilds.
-func (s *State) Presence(gID discord.GuildID, uID discord.UserID) (*gateway.Presence, error) {
-	if !s.Gateway.HasIntents(gateway.IntentGuildPresences) {
+func (s *State) Presence(
+	guildID discord.GuildID, userID discord.UserID) (*gateway.Presence, error) {
+
```

Comment:
Unnecessary indent change?

Reply:
The function header was longer than the 100 character limit, so I wrapped it.